.github/workflows/build.yml

```diff
@@ -153,6 +153,9 @@ jobs:
     steps:
       - uses: actions/checkout@v2
       - uses: jupyterlab/maintainer-tools/.github/actions/base-setup@v1
+      - name: Install missing dependencies
+        # On May 30, 2023 this wasn't included in the base try removing in future!
+        run: pip install typing_extensions
```

Comment:
I added this to fix the CI, worst case it is a no-op. But it will probably be fixed in the future with an update to maintainer-tools.